TRANSLATIONS.md

```diff
@@ -63,6 +63,8 @@ brew install dprint
 Ensure you can build the book, and that `mdbook serve` works. For this, follow
 the instructions in the [README](README.md).
 
+For complete info about installing necessary tools , refer [CONTRIBUTING.md](CONTRIBUTING.md#formatting-and-preparation)
```

Comment:
This is good, but can you take out the stuff about Gettext and `dprint` above? That way we avoid the duplication.

Reply:
Thanks , 😊
Done